woocommerce/wc-admin
# Remove task status endpoint

Fixes #7372 

Removes the task status endpoint and relies on data passed with individual tasks.

### Detailed test instructions:

1. In your console enter `localStorage.setItem( 'debug', 'wc-admin:*' );`
2. Navigate to the task list
2. On the appearance task, click the "Preview Site" button in the header
3. Note that the tracks includes the correct values for `has_homepage` and `upload_logo`
4. Note that the "homepage" and "import" steps are shown if no homepage has been created and if no products exist yet
5. Make sure the logo can still be updated without issue and persists on page refresh or under the customizer
4. On the tax task without TaxJar installed and in an automated tax supported country check that the automated tax calculation information is shown
5. Set your store to a country that does not have automated tax support and verify that it no longer shows
6. Check that all tests pass and no references to task status remain
